### ISSUE TITLE
L18 URLs are wrong

### ISSUE THREAD

**Issue by davidweichiang** [acl-org/acl-anthology#209]
Use this template to request a correction to the ACL Anthology.

Please ensure that you:

- [x] ...have read and understood the [criteria for corrections](http://aclanthology.info/corrections).
- [x] ...have included (if relevant) the ACL Anthology ID (e.g., J93-2003).

If approved, we will follow up with information about how to submit the correction.

**Comment by mjpost** [comment]
In that, they point to Anthology PDFs instead of LREC-hosted ones?

**Comment by mbollmann** [comment]
I took a guess and looked in `L18.xml`. The contents of the (currently unused) `<url>` fields there indeed look wrong.

**Comment by davidweichiang** [comment]
Exactly -- sorry I was so vague. It should be easy to fix, but I was in the middle of something else and just made this issue for later.

**Comment by mjpost** [comment]
No need to apologize, tag-and-fill-later is a good MO. The issue is clear from a look at the XML.

**Comment by nefujiangping** [comment]
It's the same with `L10`,`L12`,`L14`. However, the url of `paper` tag's attribute `href` is reachable.

**Comment by davidweichiang** [comment]
What format does the LREC metadata come to us in? Do they use EasyChair?

**Comment by knmnyn** [comment]
Hi all:

Sorry to not clarify earlier on the L??.xml format.

From the legacy Anthology, we needed to distinguish URLs that the Anthology
serves as well as the original URL for materials where the 3rd party sister
organization gives us the rights to list their materials, but may not give
us the right to host their content.

<url/> = denotes the Anthology served address for the paper
<paper href=""/>, (only used LREC) href denotes the location on the third
party site for the canonical paper.
<href/>, (only used in TACL) href denotes the location on the ACL
affiliated site for the canonical paper.

Probably we should just unify the use of <href/> and <paper href=""/> to
the former.

During my time, we had many fewer workers on the Anthology and had to
marginalize attention to sister organizations in terms of priority on
ingestions and corrections, and LREC was one of those conferences that was
affected.

ELRA recently changed their stance to allow the ACL Anthology to host their
content as well, so now they join all of the other current sister
organizations that give and indemnify ACL from listing and redistributing
their proceedings.  The copyright still belongs to the sister organization,
so any requests for reprinting and repurposing must go to their executive
committees.

Anyways, to get the the point: LREC provides no official metadata to ACL,
however Vladimir Popescu from ELRA compiled XML metadata for L18 and
provided it to me, which I have forwarded to Matt.  For L14 and earlier, I
harvested the metadata directly from the crawl of the website.  L16 is the
gap left to address.
